src/pages/Scan.vue

```diff
@@ -7,10 +7,12 @@
                 <h1>Scan a voucher code</h1>
 
                 <form id="textVoucher" v-on:submit.prevent>
-                    <transition name="fade"><div v-if="errorMessage" class="message">{{ errorMessage }}</div></transition>
-                    <label for="sponsorBox" id="lblSponsorBox" class="hidden">Sponsor code</label>
-                    <label for="voucherBox" id="lblVoucherBox" class="hidden">Voucher code</label>
-
+                    <transition name="fade">
+                        <div v-if="errorMessage" class="message">{{ errorMessage }}</div>
+                        <div v-if="queueMessage" class="queue message">{{ queueMessage }}</div>
+                    </transition>
+                    <label for="sponsorBox" id="lblSponsorBox" class="hidden">Sponsor Code</label>
```

Comment:
I thought I spotted another PR where these were lower cased. 'Sponsor code' - But not sure - just need to check consistency.